core/src/test/java/io/aiven/klaw/service/TopicControllerServiceTest.java

```diff
@@ -204,65 +185,314 @@ public void createTopicsFailure1() throws KlawException {
     when(manageDatabase.getKafkaEnvList(anyInt())).thenReturn(utilMethods.getEnvListsIncorrect1());
     when(manageDatabase.getTeamsAndAllowedEnvs(anyInt(), anyInt()))
         .thenReturn(Collections.singletonList("1"));
-
     when(mailService.getEnvProperty(anyInt(), anyString())).thenReturn("1");
 
     ApiResponse apiResponse = topicControllerService.createTopicsRequest(getFailureTopic1());
-
     assertThat(apiResponse.getResult()).isEqualTo(null);
   }
 
+  @Test
+  @Order(5)
+  public void createTopicDeleteRequestFailure1() {
```

Comment:
I know it's ugly, but probably instead of `Failure1`, we could spell the error out
```suggestion
  public void createTopicDeleteRequestFailureNotAuthorized() {
```

Reply:
Agree, updated.